presto-pinot/presto-pinot-lib/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-pinot</artifactId>
+        <version>0.229-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-pinot-lib</artifactId>
+    <name>presto-pinot-lib</name>
+    <description>Presto - Pinot Library</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto.pinot</groupId>
+            <artifactId>pinot-driver</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
```

Comment:
Let's push the commit in https://github.com/prestodb/presto-pinot-driver and do a 0.1.0 release. That will make review much easier.

Reply:
Sure. @wenleix has graciously volunteered to do that. I just filed https://github.com/prestodb/presto-pinot-driver/pull/1